js/apps/admin-ui/src/groups/Members.tsx

```diff
@@ -87,14 +88,29 @@ export const Members = () => {
   const getMembership = async (id: string) =>
     await adminClient.users.listGroups({ id: id! });
 
-  const getSubGroups = (groups: GroupRepresentation[]) => {
-    let subGroups: GroupRepresentation[] = [];
-    for (const group of groups!) {
-      subGroups.push(group);
-      const subs = getSubGroups(group.subGroups!);
-      subGroups = subGroups.concat(subs);
+  // this queries the subgroups using the new search paradigm but doesn't
+  // account for pagination and therefore isn't going to scale well
+  const getSubGroups = async (groupId: string, count: number = 0) => {
```

Comment:
We can omit the type for `count` here, as it can be inferred bu TypeScript.
```suggestion
  const getSubGroups = async (groupId: string, count = 0) => {
```